### PR TITLE
Filter openai response console logs

### PR DESCRIPTION
**Pull request by Lerri-Cofannos**

### CODE REVIEW

File: example/main.py [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 
 prompt = "What is a proompt?"
 log = get_openai_response(prompt)
-log["input"]= prompt
+log["input"] = prompt
 
 load_dotenv()
 DATABASE_URL = os.getenv("DATABASE_URL")
 if not DATABASE_URL:
-    raise EnvironmentError(f"DATABASE_URL not found: {DATABASE_URL}\nMake sure that you have a DATABASE_URL value in the .env file.")
```
Comment (CallumHemsley) [inline example/main.py:16]: why does this keep changing every PR btw?

File: sdk-python/omnilog/start_listener.py
Comment (juskek) [inline sdk-python/omnilog/start_listener.py:1]: Is there any benefit to splitting this file up to separate the classes?

File: sdk-python/test/openai_filter_test.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import unittest
+from omnilog.start_listener import OpenAIFilter
+
+
+class TestOpenAIFilter(unittest.TestCase):
+    input: str
+    result: str
+
+    def compute_result(self, name):
+        parent = logging.getLogger(name)
```
Comment (juskek) [inline sdk-python/test/openai_filter_test.py:12]: Is Omnilog being mocked by the parent or child in this case? Or am I misunderstanding?

File: example/main.py (hunk 1 above)
Comment (Lerri-Cofannos) [reply example/main.py:16]: I am not sure, but having the argument on a new line is to follow black formatting and is the correct version

File: sdk-python/omnilog/start_listener.py
Comment (Lerri-Cofannos) [reply sdk-python/omnilog/start_listener.py:1]: The benefit I see is to have files of reasonable sizes, and to separate each scope: catching logs on one side, and calling the neon database on the other.

File: sdk-python/test/openai_filter_test.py (hunk 2 above)
Comment (Lerri-Cofannos) [reply sdk-python/test/openai_filter_test.py:12]: The child is the one to receive the input, so the child is sys.stdout, and thus the parent is OmniLog. The logs propagate upwards

**Review by CallumHemsley — APPROVED** [review]

**Pull request merged**